.gitignore

```diff
@@ -133,23 +133,25 @@ iOSInjectionProject/
 ### SwiftPM ###
 Packages
 xcuserdata
-*.xcodeproj
```

Comment:
This was needed to be able to commit the scheme for the UITest. Otherwise the CI wouldn't see the custom scheme and would not be able to run UI tests.